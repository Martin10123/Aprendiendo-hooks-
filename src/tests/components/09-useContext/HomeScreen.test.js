import React from "react";
import { user } from "../fixtures/userContext";
const { mount } = require("enzyme");
const {
  default: HomeScreen,
} = require("../../../components/09-useContext/HomeScreen");
const {
  UserContext,
} = require("../../../components/09-useContext/UserContext");

describe("Pruebas en el componente HomeScreen", () => {
  const wrapper = mount(
    <UserContext.Provider value={{ user }}>
      <HomeScreen />
    </UserContext.Provider>
  );
  test("Debe mostrarse correctamente", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
