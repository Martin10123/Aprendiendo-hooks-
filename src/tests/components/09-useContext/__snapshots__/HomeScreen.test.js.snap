// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente HomeScreen Debe mostrarse correctamente 1`] = `
<div>
  <h1>
    Home Screen
  </h1>
  <hr />
  <pre>
    {
   "name": "Martin",
   "email": "martin@gmail.com"
}
  </pre>
</div>
`;
